home/models.py

```diff
@@ -295,7 +295,29 @@ def clean(self, value):
                 f"{self.MEDIA_CAPTION_MAX_LENGTH} characters, your message is "
                 f"{len(result['message'])} characters long"
             )
+
+        # find variables
+        vars_in_msg = re.findall(r"{{(.*?)}}", result["message"])
+        non_digit_variables = [var for var in vars_in_msg if not var.isdigit()]
+        if non_digit_variables != []:
```

Comment:
Usually in Python, we use the fact that lists are truthy if they are not empty, to create an if statement that's easier to read, ie. `if non_digit_variables:`